dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
-
 logging.properties=Logging
 logging.properties.desc=Controls the capture and output of log and trace messages.
 
 #
 # File size / max files
 #
 max.file.size=Maximum log file size
-max.file.size.desc=Maximum size of a log file, in megabytes, before being rolled over; a value of 0 means no limit.
+max.file.size.desc=The maximum size (in MB) that a log file can reach before it is rolled. The Liberty runtime does only size-based log rolling. To disable this attribute, set the value to 0. The maximum file size is approximate. By default, the value is 20. Note: maxFileSize does not apply to the console.logfile.
```

Comment:
Change
"Note: maxFileSize does not apply to the console.logfile."
to
"Note: maxFileSize does not apply to the console.log file."